common/utils.go

```diff
@@ -8,7 +8,7 @@ import (
 const (
 	envServices                  = "SERVICES"
```

Comment:
Where are these private consts in use? I didn't find any reference of them in other modules.